source/_integrations/duosida_ev.markdown

```diff
@@ -0,0 +1,144 @@
+---
+title: Duosida EV Charger
+description: Instructions on how to integrate Duosida EV wall chargers with Home Assistant.
+ha_category:
+  - Car
+ha_release: "2025.2"
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@americodias'
+ha_domain: duosida_ev
+ha_platforms:
+  - binary_sensor
+  - button
+  - number
+  - sensor
+  - switch
+ha_integration_type: device
+---
+
+The **Duosida EV Charger** {% term integration %} allows you to monitor and control your Duosida EV wall charger locally over your network. This integration communicates directly with the charger via TCP, without requiring cloud services.
+
+## Supported devices
+
+- Duosida SmartChargePI wall chargers (single-phase and three-phase models)
+- Other Duosida wall chargers with network connectivity on TCP port 9988
+
+## Prerequisites
+
+Before setting up this integration, ensure that:
+
+- Your Duosida charger is connected to your local network
+- You know the charger's IP address (check your router's DHCP client list)
+- TCP port 9988 is accessible on the charger from your Home Assistant instance
+
+{% tip %}
+Set a DHCP reservation in your router to keep the charger's IP address stable.
+{% endtip %}
+
+{% include integrations/config_flow.md %}
+
+## Configuration options
+
+The integration provides the following configuration options during setup:
```

Comment:
```suggestion
```
The configuration option section describes the integration options. If I understand correctly, these parameters are used during setup. They can go together with the config flow.